games/o.yaml

```
@@ -387,9 +387,10 @@
   development: sporadic
   originals:
   - Quake 3
-  repo: http://openarena.ws/svn
+  repo: https://github.com/OpenArena/engine
   type: remake
-  url: http://openarena.ws/
+  updated: 2019-08-20
+  url: http://openarena.ws/smfnews.php
```

```suggestion
  url: http://openarena.ws/
```
I would leave this unchanged because if they do change their web framework, the url will still be ok

---

Done